python/pdstools/__init__.py

```diff
@@ -44,7 +45,7 @@
     "read_ds_export",
     "setupAzureOpenAI",
     "Prediction",
-    "NBAD",
```

Comment:
@StijnKas Need help to sort out the change from NBAD to CDH_Guidelines. test_Prediction only fails because of some of this dependency stuff I think.

Reply:
CDH_Guidelines is now in the `adm` folder - but I'm not entirely sure what you're trying to access in your Predictions code, @operdeck ? Could you try `from ..adm import CDH_Guidelines` and see which methods/properties you're missing to have parity with before?